exammode/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.urls import path
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^$',
```

Comment:
The `url` function will be deprecated. `re_path` is a direct replacement.

The URL name `home` collides with another URL in `course.urls`. Why?

Reply:
The URL paths below that only contain a number look suspicious. Nothing in the URL really refers to the exammode since the the exammode URLs are included in the main aplus URL conf without any prefix word.

Reply:
Add exam key word to the beginning of each url

Reply:
Should be now fixed